### PR TITLE
Hotfix: Recent Activity Table Dates

### PR DESCRIPTION
**Pull request by alburde1**
Convert date to local date in recent activity table

### CODE REVIEW

File: src/js/components/landing/recentActivity/RecentActivityTable.jsx [hunk 1]
```diff
@@ -98,6 +98,24 @@ export default class RecentActivityTable extends React.Component {
...
+        let month = localDate.getMonth()+1;
+        if(month < 10){
+            month = "0"+month;
+        }
+        let day = localDate.getDate();
+        if(day <10){
+            day = "0"+day;
+        }
+        return year + "-" + month + "-" + day;
+    }
```
Comment (kkuratsu) [inline src/js/components/landing/recentActivity/RecentActivityTable.jsx]: Can you check indentation here?

File: src/js/components/landing/recentActivity/RecentActivityTable.jsx [hunk 2]
```diff
@@ -98,6 +98,24 @@ export default class RecentActivityTable extends React.Component {
...
+        const localDate = new Date(tmpDate.toString())
+        
+        // format date as YYYY-MM-DD
+        const year = localDate.getFullYear()
+        let month = localDate.getMonth()+1;
+        if(month < 10){
+            month = "0"+month;
+        }
+        let day = localDate.getDate();
+        if(day <10){
```
Comment (kkuratsu) [inline src/js/components/landing/recentActivity/RecentActivityTable.jsx]: I would add a space between `if (day < 10){`.

File: src/js/components/landing/recentActivity/RecentActivityTable.jsx [hunk 3]
```diff
@@ -98,6 +98,24 @@ export default class RecentActivityTable extends React.Component {
...
     }
 
+	convertToLocalDate(dateToConvert) {
+        // convert date to local date (toString converts it to whatever the local time is but doesn't allow formatting)
+        const tmpDate = new Date(dateToConvert + " UTC");
+        const localDate = new Date(tmpDate.toString())
+        
+        // format date as YYYY-MM-DD
+        const year = localDate.getFullYear()
+        let month = localDate.getMonth()+1;
```
Comment (kkuratsu) [inline src/js/components/landing/recentActivity/RecentActivityTable.jsx]: Spacing needed here `localDate.getMonth() + 1`.

File: src/js/components/landing/recentActivity/RecentActivityTable.jsx [hunk 4]
```diff
@@ -98,6 +98,24 @@ export default class RecentActivityTable extends React.Component {
...
+        if(month < 10){
+            month = "0"+month;
```
Comment (kkuratsu) [inline src/js/components/landing/recentActivity/RecentActivityTable.jsx]: Spacing needed here `"0" + month`

File: src/js/components/landing/recentActivity/RecentActivityTable.jsx [hunk 5]
```diff
@@ -98,6 +98,24 @@ export default class RecentActivityTable extends React.Component {
...
+        }
+        let day = localDate.getDate();
+        if(day <10){
+            day = "0"+day;
```
Comment (kkuratsu) [inline src/js/components/landing/recentActivity/RecentActivityTable.jsx]: Spacing needed here `"0" + day`

File: src/js/components/landing/recentActivity/RecentActivityTable.jsx [hunk 6]
```diff
@@ -98,6 +98,24 @@ export default class RecentActivityTable extends React.Component {
...
 
+	convertToLocalDate(dateToConvert) {
+        // convert date to local date (toString converts it to whatever the local time is but doesn't allow formatting)
+        const tmpDate = new Date(dateToConvert + " UTC");
+        const localDate = new Date(tmpDate.toString())
+        
+        // format date as YYYY-MM-DD
+        const year = localDate.getFullYear()
+        let month = localDate.getMonth()+1;
+        if(month < 10){
```
Comment (kkuratsu) [inline src/js/components/landing/recentActivity/RecentActivityTable.jsx]: Space needed here `if (month < 10){`

File: src/js/components/landing/recentActivity/RecentActivityTable.jsx (hunk 1 above)
Comment (alburde1) [inline src/js/components/landing/recentActivity/RecentActivityTable.jsx]: Indentation is fine, just checked

**Pull request merged**